data/json/materials.json

```diff
@@ -522,6 +522,63 @@
         "density" : 30
     },{
         "type" : "material",
+        "ident" : "carbon_steel",
+        "name" : "High Carbon Steel",
+        "bash_resist" : 6,
+        "cut_resist" : 8,
+        "bash_dmg_verb" : "dented",
+        "cut_dmg_verb" : "scratched",
+        "dmg_adj" : [
+            "marked",
+            "dented",
+            "smashed",
+            "shattered"
+        ],
+        "acid_resist" : 7,
+        "elec_resist" : 0,
+        "fire_resist" : 3,
+        "chip_resist" : 30, 
+        "density" : 30 
+    },{
+        "type" : "material",
+        "ident" : "budget_steel",
+        "name" : "budget steel",
+        "bash_resist" : 4,
+        "cut_resist" : 2,
```

Comment:
Is this budget steel supposed to be closer to wood than to iron? Iron is 4/4, even cheap steel should not go below that.
Why lowered cut resistance?